plugins/nanospell/plugin.js

```diff
@@ -18,769 +18,774 @@
-                });
-
-                editorCheckDirty.resetDirty = CKEDITOR.tools.override(editorCheckDirty.resetDirty, function (org) {
-                    return function () {
-                        this._.previousValue = clearAllSpellCheckingSpansFromString(this.getSnapshot());
-                    };
-                });
-            }
-
-
-        },
-        addRule: function (editor) {
-            var dataProcessor = editor.dataProcessor,
-                htmlFilter = dataProcessor && dataProcessor.htmlFilter,
-                pathFilters = editor._.elementsPath && editor._.elementsPath.filters,
-                dataFilter = dataProcessor && dataProcessor.dataFilter,
-                removeFormatFilter = editor.addRemoveFormatFilter,
-                pathFilter = function (element) {
-                    if (element.hasClass('nanospell-typo')) {
-                        return false;
-                    }
-                },
-                removeFormatFilterTemplate = function (element) {
-                    var result = true;
-
-                    if (element.hasClass('nanospell-typo')) {
-                        result = false;
-                    }
-
-                    return result;
-                };
-
-            if (pathFilters) {
-                pathFilters.push(pathFilter);
-            }
-
-            if (dataFilter) {
-                var dataFilterRules = {
-                    elements: {
-                        span: function (element) {
-
-                            var scaytState = element.hasClass('nanospell-typo');
-
-                            if (scaytState) {
-                                delete element.name;
-                            }
-
-                            return element;
-                        }
-                    }
-                };
-
-                dataFilter.addRules(dataFilterRules);
-            }
-
-            if (htmlFilter) {
-                var htmlFilterRules = {
-                    elements: {
-                        span: function (element) {
-
-                            var scaytState = element.hasClass('nanospell-typo');
-
-                            if (scaytState) {
-                                delete element.name;
-                            }
-
-                            return element;
-                        }
-                    }
-                };
-
-                htmlFilter.addRules(htmlFilterRules);
-            }
-
-            if (removeFormatFilter) {
-                removeFormatFilter.call(editor, removeFormatFilterTemplate);
-            }
-        }
-
-    });
+	'use strict';
+
+	var maxRequest = 200;
+	var editorHasFocus = false;
+	var spell_delay = 250;
+	var spell_fast_after_spacebar = true;
+	var settings_path;
+	var state = false;
+	var lang = "en";
+	var locale = {
+		ignore: "Ignore",
+		learn: "Add To Personal Dictionary",
+		nosuggestions: "( No Spelling Suggestions )"
+	};
+	var spellcache = [];
```

Comment:
I would expect these to be camelCase to match javascript style, or underscore_case to match the hodpodge of style used above.